tests/containers/test_fortifw.py

```diff
@@ -57,6 +57,19 @@ def test_deobfuscate_firmware_file(header, is_gzipped):
     assert ff.size == len(FIRMWARE_HEADER)
 
 
+def test_gzip_trailer():
```

Comment:
```suggestion
def test_gzip_trailer() -> None:
```
Can you also add these to the other tests?

Reply:
done